src/main/java/streambms/ConsoleInput.java

```diff
@@ -0,0 +1,18 @@
+package streambms;
+
+import java.util.Scanner;
+
+public class ConsoleInput implements IInputType
+{
+
+  @Override
+  public String getInput()
+  {
+    System.out.println("Please enter required battery and charging parameters: ");

```

Comment:
In production, this print will go to the Receiver, due to the piping. Separate your test/demo code and production code.